src/topics/doc/security.md

```diff
@@ -15,68 +15,36 @@ collaboration with the app implementation, since the app must be allowed
 privileged access to the data in order to function.
 
 ### Threats
-The threats that must be controlled in collaboration with a clone engine are a
-subset of the threats to the app. These include threats to:
-- *Confidentiality*: reading of data to steal information
-- *Integrity*: modification of data to mislead information consumers
-- *Availability*: prevention of normal function for sabotage or coercion
+The threats that must be controlled in collaboration with a clone engine are a subset of the threats to the app. These include threats to *Confidentiality*, *Integrity* and *Availability*.
 
 The attack surface of an engine generally comprises:
 - The local device storage being used by the engine
 - The network between the engine and other clones
 - The clone API presented to the app
 
 ### Trust Model
-A decentralised data store does not have a privileged, trusted central authority
-(although the trust model of such a central authority may never have been as
-straightforward as it seemed, as evidenced by damaging data leakages by such
-authorities). Since an app must be free to decide its own security model, and
-since the **m-ld** engine is not itself afforded any special privilege because
-of its deployment, an engine *trusts the app* in principle. The consequences of
-this are explained in the sections below.
-
-### Authentication & Authorisation
-It is the app's responsibility to authenticate and authorise its users. For the
-reason above, and unlike some centralised data management systems, **m-ld** does
-not have a first-class 'user' concept with special semantics. This includes any
-notion of credentials, such as passwords. (Note that this does not prevent an
-app from storing user information in **m-ld**, so long as it uses suitable access
-controls.)
-
-This means an app is free to authenticate its users by any chosen means, such as
-device-native login, or using a third-party single sign-on system. The app
-should gate access, using the authentication, to its functions which access the
-**m-ld** engine.
-
-> 🚧 In future, **m-ld** will allow an app to negotiate a 'local key' credential
-> that the engine can use to:
-> - confirm the identity of the app instance, for example after a re-start
-> - selectively encrypt data in storage and on the network (see below)
-> - identify and suppress malware (see below)
->
-> A specification document for future security features will shortly be
-> available in this portal. Please [feed-back](/hello/) any specific concerns
-> you have.
+An app using **m-ld** is free to decide its own security model. Because the **m-ld** engine is embedded in the app, it not itself afforded any special privilege because of its deployment. In particular, a **m-ld** domain does not inherently have a privileged, trusted central authority. However, an app may choose to deploy **m-ld** clones to its own trusted service or data tier.
```

Comment:
Still needs fixing....

Reply:
GitHub's interface for this breaks down slightly... it is fixed, but if you click on the filename you only see the commit which you commented on, not the latest! Go to the top and select "Show changes since your last review" (but unfortunately that makes the comment disappear 🤷‍♂️).